easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb

```diff
@@ -36,9 +36,16 @@ dependencies = [
 
 local_default_search_engine = 'RMBlast'
 
-preinstall_cmd = './configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" '
-preinstall_cmd += '-hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" '
-preinstall_cmd += '-default_search_engine %s' % local_default_search_engine.lower()
+#preinstall_cmd = './configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" '
+#preinstall_cmd += '-hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" '
+#preinstall_cmd += '-default_search_engine %s' % local_default_search_engine.lower()
+
+local_config_command = 'cd %(installdir)s &&'
+local_config_command += './configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" '
+local_config_command += '-hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" '
+local_config_command += '-default_search_engine %s' % local_default_search_engine.lower()
+
+postinstallcmds = [ local_config_command, ]
```

Comment:
@wdlingit welcome to EasyBuild, and thanks for reporting the issue and even sending a patch.

This looks fine overall, though oughtn't this line to be `preinstall_cmd`?

Reply:
@wdlingit Please also edit the PR title to briefly describe what this change is fixing, so we have a meaningful entry in the release notes (cfr. https://docs.easybuild.io/en/latest/Release_notes.html)

Reply:
@verdurin If you are referring to the configure step, actually it is for configuring database for the program but not configuring the program. This is why we better to execute it at the installdir and put it in postinstallcmds.

@boegel Just edited the title. Please let me know if you think it should be more meaningful than the current version.

Reply:
@verdurin I just tried to set `preinstall_cmd` with the `local_config_command`. Seems it would be failed because the `installdir` was not created when executing `preinstall_cmd`. So it seems better to put the configure command in `postinstallcmds`.

```
wdlin@comp15:~/easybuild-easyconfigs$ git diff easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb
diff --git a/easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb b/easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb
index 7b7bd2f8ef..f8ae4d30cc 100644
--- a/easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb
+++ b/easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb
@@ -36,16 +36,10 @@ dependencies = [

 local_default_search_engine = 'RMBlast'

-#preinstall_cmd = './configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" '
-#preinstall_cmd += '-hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" '
-#preinstall_cmd += '-default_search_engine %s' % local_default_search_engine.lower()
-
-local_config_command = 'cd %(installdir)s &&'
-local_config_command += './configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" '
-local_config_command += '-hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" '
-local_config_command += '-default_search_engine %s' % local_default_search_engine.lower()
-
-postinstallcmds = [ local_config_command, ]
+preinstall_cmd = 'cd %(installdir)s &&'
+preinstall_cmd += './configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" '
+preinstall_cmd += '-hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" '
+preinstall_cmd += '-default_search_engine %s' % local_default_search_engine.lower()

 fix_perl_shebang_for = ['RepeatMasker']

wdlin@comp15:~/easybuild-easyconfigs$ eb -r -f RepeatMasker-4.1.2-p1-foss-2020b.eb
== Temporary log file in case of crash /tmp/eb-xcnm6vuf/easybuild-133_2ik2.log
== resolving dependencies ...
== processing EasyBuild easyconfig /home/wdlin/easybuild-easyconfigs/easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb
== building and installing RepeatMasker/4.1.2-p1-foss-2020b...
== fetching files...
== ... (took 1 secs)
== creating build dir, resetting environment...
== unpacking...
== ... (took 13 secs)
== patching...
== preparing...
== ... (took 2 secs)
== configuring...
== building...
== testing...
== installing...
== FAILED: Installation ended unsuccessfully (build directory: /home/wdlin/.local/easybuild/build/RepeatMasker/4.1.2-p1/foss-2020b): build failed (first 300 chars): cmd "cd /home/wdlin/.local/easybuild/software/RepeatMasker/4.1.2-p1-foss-2020b &&./configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" -hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" -default_search_engine rmblast" exited with exit code 127 and output:
/bin/bash: (took 18 secs)
== Results of the build can be found in the log file(s) /tmp/eb-xcnm6vuf/easybuild-RepeatMasker-4.1.2-p1-20220420.134409.YJfGu.log
ERROR: Build of /home/wdlin/easybuild-easyconfigs/easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.1.2-p1-foss-2020b.eb failed (err: 'build failed (first 300 chars): cmd "cd /home/wdlin/.local/easybuild/software/RepeatMasker/4.1.2-p1-foss-2020b &&./configure -perlbin "$EBROOTPERL/bin/perl" -trf_prgm "$EBROOTTRF/bin/trf" -hmmer_dir "$EBROOTHMMER/bin" -rmblast_dir "$EBROOTRMBLAST/bin" -default_search_engine rmblast" exited with exit code 127 and output:\n/bin/bash:')
```